MosipNexus/run_update.py

```
@@ -92,8 +94,28 @@ def run() -> None:
+        from ingestion.store import ingest_json_file
+
+        ingest_json_file(
+            ESIGNET_FILE,
+            ESIGNET_COLLECTION,
+            embeddings,
+            source_type="esignet",
+        )
+
+        state.setdefault("esignet", {})["last_run"] = now_iso()
+
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)